presto-main/src/main/java/com/facebook/presto/sql/planner/SubqueryPlanner.java

```diff
@@ -320,10 +325,76 @@ private PlanBuilder appendQuantifiedComparisonApplyNode(PlanBuilder subPlan, Qua
                         return appendQuantifiedComparisonApplyNode(subPlan, rewrittenAny, correlationAllowed);
                 }
                 break;
+
+            case LESS_THAN:
+            case LESS_THAN_OR_EQUAL:
+            case GREATER_THAN:
+            case GREATER_THAN_OR_EQUAL:
+                return planQuantifiedOrderable(subPlan, quantifiedComparison, correlationAllowed);
         }
         throw new SemanticException(NOT_SUPPORTED, quantifiedComparison, "Quantified comparison with the given quantifier and comparison type is not supported");
     }
 
+    private PlanBuilder planQuantifiedOrderable(PlanBuilder subPlan, QuantifiedComparisonExpression quantifiedComparison, boolean correlationAllowed)
+    {
+        checkArgument(quantifiedComparison.getComparisonType() == LESS_THAN || quantifiedComparison.getComparisonType() == LESS_THAN_OR_EQUAL ||
```

Comment:
You can do `EnumSet.of(LESS_THAN, LESS_THAN_OR_EQUAL, ...).contains(quantifiedComparison.getComparisonType())`